EmarsysInbox/EmarsysInbox.storyboard

```diff
@@ -187,11 +187,11 @@
                                             <rect key="frame" x="0.0" y="0.0" width="414" height="644"/>
                                             <autoresizingMask key="autoresizingMask"/>
                                             <subviews>
-                                                <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="logo" translatesAutoresizingMaskIntoConstraints="NO" id="sfl-0C-olx">
-                                                    <rect key="frame" x="0.0" y="0.0" width="414" height="290"/>
+                                                <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleToFill" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="logo" translatesAutoresizingMaskIntoConstraints="NO" id="sfl-0C-olx">
```

Comment:
we may want to keep it `scaleAspectFit`, otherwise image may be 'stretched' unexpectedly

Reply:
or `scaleAspectFill`